easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb

```diff
@@ -0,0 +1,74 @@
+# easybuild easyconfig
+#
+# Graphviz has moved to gitlab and it is no longer possible to get tagged
+# versions.  There is an open ticket this issue.
+
+easyblock = 'ConfigureMake'
+
+name = 'Graphviz'
+version = '2.40.1'
+
+homepage = 'http://www.graphviz.org/'
+description = """Graphviz is open source graph visualization software. Graph
+ visualization is a way of representing structural information as diagrams of
+ abstract graphs and networks. It has important applications in networking,
+ bioinformatics,  software engineering, database and web design, machine
+ learning, and in visual interfaces for other technical domains."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://graphviz.gitlab.io/pub/graphviz/stable/SOURCES/']
+sources = ['graphviz.tar.gz']
+checksums = ['ca5218fade0204d59947126c38439f432853543b0818d9d728c589dfe7f3a421']
+
+python = "Python"
+pythonversion = '2.7.15'
+
+dependencies = [
+    ('cairo', '1.14.12'),
+    ('expat', '2.2.5'),
+    ('freetype', '2.9.1'),
+    ('fontconfig', '2.13.0'),
+    ('Ghostscript', '9.23'),
+    ('GObject-Introspection', '1.54.1', '-Python-2.7.15'),
+    ('GTS', '0.7.6'),
+    ('Java', '1.8', '', True),
+    ('libpng', '1.6.34'),
+    ('Pango', '1.42.4'),
+    ('Perl', '5.28.0'),
+    ('Qt5', '5.10.1'),
+    ('Tcl', '8.6.8'),
+    ('zlib', '1.2.11'),
+]
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('SWIG', '3.0.12', '-Python-3.6.6'),
```

Comment:
@fizwit You can't mix Python 2.x and Python 3.x in a single dependency tree, so this should be changed to use the Python 2.7.15 version of SWIG:

```python
('SWIG', '3.0.12', '-Python-2.7.15'),
```

(that's also why the tests currently fail due to a conflict problem)